src/metadata/metadata_handler.h

```diff
@@ -123,6 +124,7 @@ static constexpr auto mt_keys = std::array<std::pair<metadata_fields_t, const ch
     { M_ARTIST, "upnp:artist" },
     { M_ALBUM, "upnp:album" },
     { M_DATE, "dc:date" },
+    { M_CREATION_DATE, "dc:date" },
```

Comment:
You need to avoid overwriting here also. I recommend `upnp:recordedStartDateTime`. There's no real equivalent in upnp spec. I'm not sure whether there is some sort of property mapping in UPNP_XML

Reply:
> You need to avoid overwriting here also. I recommend `upnp:recordedStartDateTime`. There's no real equivalent in upnp spec. I'm not sure whether there is some sort of property mapping in UPNP_XML

I found [DCMI Metadata Terms](https://www.dublincore.org/specifications/dublin-core/dcmi-terms/) linked from [/UPnP-av spec](http://upnp.org/specs/av/UPnP-av-ContentDirectory-v1-Service.pdf).
Refer to this, `dc:created` is suitable.
I fixed it.